platform/src/default.rs

```diff
@@ -0,0 +1,11 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{CertChunk, Platform, PlatformError};
+
+pub struct DefaultPlatform;
+
+impl Platform for DefaultPlatform {
+    fn get_certificate_chain(_offset: u32, _size: u32) -> Result<CertChunk, PlatformError> {
+        Err(PlatformError::NotImplemented)
```

Comment:
I would recommend generating a test cert (with OpenSSL or something) and putting it in a buffer so the things that depend on this can be tested. Something like [this](https://stackoverflow.com/questions/11992036/how-do-i-create-an-ecdsa-certificate-with-the-openssl-command-line).

Then you can hex dump the file (with e.g. `xxd` and paste it in the code as a `u8` array.

At some point, would be good to make that cert have the same public key as the alias key cert, but that can come later.